lib/rails_utils/version.rb

```diff
@@ -1,5 +1,5 @@
 # frozen_string_literal: true
 
 module RailsUtils
-  VERSION = "5.0.0"
+  VERSION = "6.0.0"
```

Comment:
What is new in this major version bump to 6.0.0?

Reply:
updated the CHANGELOG accordingly